test/functional/fixtures/api/es-next/selector/testcafe-fixtures/selector-test.js

```diff
@@ -481,6 +481,95 @@ test('Selector "withText" method', async () => {
     expect(await elWithClass('idxEl').withText('element 1.').id).eql('el1');
 });
 
+test('Selector "withAttr" method', async t => {
+    // String selector
+    // string attr name
+    let selector = Selector('div').withAttr('data-store');
+
+    let el = await selector();
+
+    await t.expect(el.id).eql('attr1');
```

Comment:
Why not use Shorthand properties in this test, like: `await t.expect(selector.id).eql('attr1')`